src/pages/pass-finalisation/cat-be/pass-finalisation.cat-be.page.scss

```diff
@@ -0,0 +1,12 @@
+page-pass-finalisation {
```

Comment:
You need to change this to `page-pass-finalisation-cat-be-page` else we will lose the styling